### PR TITLE
python-pytest: update to version 6.0.1

### PR DESCRIPTION
**Pull request by ja-pa**
Maintainer: me
Compile tested: Turris Omnia (TOS5), OpenWrt master
Run tested: Turris Omnia (TOS5), OpenWrt master

Description:
This PR updates pytest to version 6.0.1

It also adds **python-toml**, **python-iniconfig** packages which are required by 6.0.x version and updates **python-py** package.

Run tested by running test set:
```
platform linux -- Python 3.8.3, pytest-6.0.1, py-1.9.0, pluggy-0.13.1
rootdir: /ccc/pytest, configfile: pyproject.toml, testpaths: testing
plugins: xdist-1.34.0
collected 2727 items / 2 errors / 2725 selected                                                                                                                                                            
```

### CODE REVIEW

**Comment by BKPepe** [comment]
Why this is still a draft?

**Comment by ja-pa** [comment]
> Why this is still a draft?

I was reworking it to 6.0.1 update which adds 2 new packages.

**Comment by ja-pa** [comment]
cc @jefferyto @commodo

**Comment by ja-pa** [comment]
ping @jefferyto @commodo

**Comment by commodo** [comment]
apologies for the delay here;

this looks good from my side;

**Comment by ja-pa** [comment]
> this looks good from my side;

NP, thanks!

File: lang/python/python-pytest/Makefile [hunk 1]
```diff
@@ -41,6 +41,8 @@ define Package/python3-pytest
 	+python3-wcwidth \
 	+python3-decimal \
 	+python3-logging \
+	+python3-toml \
+	+python3-iniconfig \
```
Comment (jefferyto) [inline lang/python/python-pytest/Makefile:45]: wcwidth has been [removed](https://github.com/pytest-dev/pytest/blob/6.0.1/doc/en/changelog.rst#trivialinternal-changes-1) as a dependency.

Also, in addition to decimal, logging and urllib, [codecs](https://github.com/pytest-dev/pytest/blob/6.0.1/src/_pytest/_io/wcwidth.py#L1), [pydoc](https://github.com/pytest-dev/pytest/blob/6.0.1/src/_pytest/doctest.py#L171) and [unittest](https://github.com/pytest-dev/pytest/blob/6.0.1/src/_pytest/unittest.py#L68) are needed. I think at this point would be better to replace python3-light and the other stdlib packages with python3.

File: lang/python/python-toml/Makefile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+PYPI_NAME:=toml
+PKG_HASH:=926b612be1e5ce0634a2ca03470f95169cf16f939018233a670519cb4ac58b0f
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_PARALLEL:=0
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=setuptools-scm
```
Comment (jefferyto) [inline lang/python/python-toml/Makefile:22]: setuptools-scm [does not appear](https://github.com/uiri/toml/search?q=setuptools-scm&unscoped_q=setuptools-scm) to be needed.

File: lang/python/python-pytest/Makefile (hunk 1 above)
Comment (ja-pa) [reply lang/python/python-pytest/Makefile]: Replace python-light, python3-urllib, python3-wcwidth, python3-decimal, python3-logging with python3 and add dependencies to python-py

File: lang/python/python-toml/Makefile (hunk 2 above)
Comment (ja-pa) [reply lang/python/python-toml/Makefile]: Removed

**Comment by ja-pa** [comment]
@jefferyto Thx for feedback, I fixed dependencies and rebase it to master.

File: lang/python/python-toml/Makefile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+PKG_RELEASE:=1
+
+PYPI_NAME:=toml
+PKG_HASH:=926b612be1e5ce0634a2ca03470f95169cf16f939018233a670519cb4ac58b0f
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_PARALLEL:=0
```
[the remark this answers is not in the document]
Comment (ja-pa) [reply lang/python/python-toml/Makefile:21]: Thx, it was left over, Removed

**Pull request merged**